src/Components/v2/AutoScalingImageLink.tsx

```diff
@@ -0,0 +1,126 @@
+import { color, Flex, Serif, SerifSize } from "@artsy/palette"
+import React from "react"
+import styled from "styled-components"
+import { space, SpaceProps } from "styled-system"
+
+export const AutoScalingImageLink: React.FC<AutoScalingImageLinkProps> = ({
+  href,
+  imageUrl,
+  children,
+  fontSize,
+  maxWidth,
+  maxHeight,
+  ...rest
+}) => {
+  return (
+    <LinkWithBackground href={href} imageUrl={imageUrl} {...rest}>
+      <Flex
+        justifyContent="center"
+        alignItems="center"
+        position="relative"
+        width="100%"
+      >
+        <LinkText size={fontSize} textAlign="center" p={1}>
+          {children}
+        </LinkText>
+        <PlaceholderImagesForContainerHeightAndWidth
+          maxWidth={maxWidth}
+          maxHeight={maxHeight}
+        />
+      </Flex>
+    </LinkWithBackground>
+  )
+}
+
+interface AutoScalingImageLinkProps
+  extends SpaceProps,
+    LinkWithBackgroundProps,
+    PlaceholderImagesForContainerHeightAndWidthProps {
+  href: string
+  fontSize?: SerifSize
+}
+
+AutoScalingImageLink.defaultProps = {
+  fontSize: "5t",
+}
+
+const LinkText = styled(Serif)`
+  position: absolute;
+  vertical-align: middle;
+  text-align: center;
+`
+
+// This placeholder gives the container the correct height/width/aspect ratio at each breakpoint.
+//   It is hidden by CSS, but it allows the background image to scale correctly
+//   based on the available width.
+const PlaceholderImagesForContainerHeightAndWidth: React.FC<
```

Comment:
#minor This is a singular item here, yeah? So maybe just `PlaceholderImageForContainerHeightAndWidth`?

Reply:
Yeah, that makes sense. Historically, this thing was rendering one of multiple images - I should have updated it when it became only one. I'll fix!